src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -486,6 +486,39 @@ public unsafe bool CursorGrabbed
             }
         }
 
+        /// <inheritdoc />
+        public int CurrentMonitorPpi
+        {
+            get
+            {
+                int widthInMm;
+                int heightInMm;
+                int widthInPixels;
+                int heightInPixels;
+                float horizontalPpi;
+                float verticalPpi;
+
+                unsafe
+                {
+                    GraphicsLibraryFramework.Monitor* monitor =
+                        CurrentMonitor.ToUnsafePtr<GraphicsLibraryFramework.Monitor>();
+                    VideoMode* videoMode;
+
+                    GLFWProvider.GLFW.Value.GetMonitorPhysicalSize(monitor, out widthInMm, out heightInMm);
+                    videoMode = GLFWProvider.GLFW.Value.GetVideoMode(monitor);
+
+                    widthInPixels = videoMode->Width;
+                    heightInPixels = videoMode->Height;
+                }
+
+                horizontalPpi = (float)widthInPixels / ((float)widthInMm / 25.4f);
+                verticalPpi = (float)heightInPixels / ((float)heightInMm / 25.4f);
```

Comment:
Can this calculation be verified via Wikpedia or any other good source?

Reply:
I'd suggest to create a private method which returns the platform defaults via `RuntimeInformation
                                               .IsOSPlatform(OSPlatform.Windows)` and co..